### ISSUE TITLE
Implement superoperator representation transformation

### ISSUE THREAD

**Issue by labay11** [qutip/QuantumToolbox.jl#270]
### Problem Description

Implement functions to convert from and to kraus, choi, chi and superoperator.

### Proposed Solution

All the functions are already implemented in qutip in https://github.com/qutip/qutip/blob/master/qutip/core/superop_reps.py

Additionally, in metrics we could implement the process fidelity as in https://github.com/qutip/qutip/blob/master/qutip/core/metrics.py#L152

**Comment by ytdHuang** [comment]
We actually have a previous PR #181 a few months ago.
But we don't have time for developing this functionality so far.
Would you like to contribute ?

**Comment by labay11** [comment]
Yes, if there's no hurry for implementing this I can try to do it

**Comment by ytdHuang** [comment]
Okay, I suggest that you try it after PR #266 is merged.
Because there's a huge change of the code in this PR.

**Comment by ytdHuang** [comment]
The features I mentioned in PR #271 might be necessary for this PR.
